scripts/get-venues-chats.ts

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env node -r esm -r ts-node/register
+
+import { writeFile } from "fs";
+import { resolve } from "path";
+
+import type firebase from "firebase/app";
+import admin from "firebase-admin";
+
+import { ChatMessageType, PollQuestion, PollVote } from "../src/types/chat";
+import { formatDate, formatTimeLocalised } from "../src/utils/time";
+
+import { initFirebaseAdminApp, makeScriptUsage } from "./lib/helpers";
+
+const usage = makeScriptUsage({
+  description: "Retrieve 'chats' details (in CSV format) ",
+  usageParams: "PROJECT_ID VENUE_IDS [CREDENTIAL_PATH]",
+  exampleParams:
+    "co-reality-map venueId,venueId2,venueIdN [theMatchingAccountServiceKey.json]",
+});
+
+const [projectId, venueIds, credentialPath] = process.argv.slice(2);
+
+if (!projectId || !venueIds) {
+  usage();
+}
+
+const venueIdsArray = venueIds.split(",");
+
+if (venueIdsArray.length > 10) {
+  console.error(
+    "Error: This script can only handle up to 10 venueIds at once at the moment."
+  );
+  console.error("  venueIdsArray.length :", venueIdsArray.length);
+  process.exit(1);
+}
+
+const app = initFirebaseAdminApp(projectId, {
+  credentialPath: credentialPath
+    ? resolve(__dirname, credentialPath)
+    : undefined,
+});
+
+const calculateVotePercentage = (item: PollQuestion, votes: PollVote[]) =>
+  votes.length
+    ? Math.floor(
+        (votes.filter(({ questionId }) => questionId === item.id).length /
+          votes.length) *
+          100
+      )
+    : 0;
```

Comment:
Let's unify this method and calculateVotePercentage in `src/components/molecules/ChatPoll/ChatPoll.tsx`